src/views/playground/Playgrounds/PlaygroundMemberChannel.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react'
+import { useMember } from '@/hooks'
+import { Button, RadioButton } from '@/shared/components'
+
+const mockMember = {
+  id: '1234',
+  activeChannelId: '5678',
+}
+const channels = ['103123213', '1230123021', '123912399132']
+
+const PlaygroundMemberChannel = () => {
+  const { member, setMember, removeMember, setActiveChannel } = useMember()
+
+  const [selected, setSelected] = useState<string | number>('')
+  const [memberString, setMemberString] = useState('')
+
+  const handleActiveChannelChange: (e: React.MouseEvent<HTMLElement>) => void = (e) => {
+    const element = e.currentTarget as HTMLInputElement
+    setSelected(element.value)
+    setActiveChannel(element.value)
+  }
```

Comment:
```suggestion
  const handleActiveChannelChange = (e: React.MouseEvent<HTMLInputElement>) => {
    const element = e.currentTarget
    setSelected(element.value)
    setActiveChannel(element.value)
  }
```
Not sure why you did that. I think what I suggested here will also work